pkg/stream/kafka/connection/config.go

```diff
@@ -0,0 +1,11 @@
+package connection
+
+type Conf struct {
```

Comment:
In the gosoline ecosystem, we call these structs just `Settings`.

Reply:
Pushed.